packages/create-cloudflare/templates-experimental/astro/templates/js/wrangler.toml

```diff
@@ -1,12 +0,0 @@
-#:schema node_modules/wrangler/config-schema.json
-name = "<TBD>"
-compatibility_date = "<TBD>"
-compatibility_flags = ["nodejs_compat_v2"]
-main = "./dist/_worker.js/index.js"
-assets = { directory = "./dist", binding = "ASSETS" }
-
-# Workers Logs
-# Docs: https://developers.cloudflare.com/workers/observability/logs/workers-logs/
-# Configuration: https://developers.cloudflare.com/workers/observability/logs/workers-logs/#enable-workers-logs
```

Comment:
I'm noticing that we don't get comments in the new json files, are we ok dropping these? or should they be copied over too?

Reply:
I've changed C3 to inject those in when modifying the config file, to make sure they're consistent and only need updated in one place

Reply:
I see, actually you removed this log altogether and what you inject is just the simple link to the config docs, right?

sounds good to me, also sorry I missed the conversation above about comments 👍